template.php

```diff
@@ -425,3 +425,18 @@ function _campaignion_foundation_exclude_block_from_contextual_links($module) {
   ];
   return in_array($module, $exclude);
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter() for webform_client_form().
+ *
+ * Provide the form step information in Drupal.settings for the behavior
+ * JavaScripts to pick up.
+ */
+function campaignion_foundation_form_webform_client_form_alter(&$form, &$form_state, $form_id) {
+  $settings['campaignion_foundation']['webform'] = [
+    'total_steps' => $form_state['webform']['page_count'],
+    'current_step' => $form_state['webform']['page_num'],
+    'last_completed_step' => isset($form_state['webform']['page_visited']) ? $form_state['webform']['page_visited'] : 0,
```

Comment:
We’re already using `??` in this theme, so I guess it would be ok to use it here too…?